tutorials/manual/build.cmake

```diff
@@ -1,6 +1,5 @@
 execute_process(
-     COMMAND pdflatex tftutorials.tex && pdflatex tftutorials.tex && bibtex tftutorials.aux && bibtex tftutorials.aux && pdflatex
-tftutorials.tex && pdflatex tftutorials.tex
+     COMMAND pdflatex tftutorials.tex && pdflatex tftutorials.tex && bibtex tftutorials.aux && bibtex tftutorials.aux && pdflatex tftutorials.tex && pdflatex tftutorials.tex
      )
```

Comment:
All I know is that the manual was not getting run through latex the last few times to resolve references (which was what was posted on figshare).  I could reproduce that problem on a mac by running the original build.make commands and it was fixed by removing the carriage return.  Bottom line,  the file is fine, it’s just not obvious that it’s being compiled enough times.  

Anyway, not sure how the manual was being compiled in the first place but if it’s done by hand it should be fine and then posted on figshare.

m

> On Feb 2, 2016, at 5:46 PM, Cian Wilson <notifications@github.com> wrote:
> 
> In tutorials/manual/build.cmake <https://github.com/TerraFERMA/TerraFERMA/pull/28#discussion_r51649300>:
> 
> > @@ -1,6 +1,5 @@
> >  execute_process(
> > -     COMMAND pdflatex tftutorials.tex && pdflatex tftutorials.tex && bibtex tftutorials.aux && bibtex tftutorials.aux && pdflatex
> > -tftutorials.tex && pdflatex tftutorials.tex
> > +     COMMAND pdflatex tftutorials.tex && pdflatex tftutorials.tex && bibtex tftutorials.aux && bibtex tftutorials.aux && pdflatex tftutorials.tex && pdflatex tftutorials.tex
> >       )
> This change makes no difference on ubuntu. Does it really matter on macs?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/TerraFERMA/TerraFERMA/pull/28/files#r51649300>.
> 

-------------------------------------------------
Marc Spiegelman
Dept of Earth and Env Sciences
Dept of App. Physics and App. Math
Columbia University
-------------------------------------------------